doc/nrf/includes/tfm.txt

```diff
@@ -1,3 +1,4 @@
-When built for the ``nrf5340dk_nrf5340_cpuapp_ns`` target, the sample is configured to compile and run as a non-secure application.
+When built for the ``_ns`` build targets, the sample is configured to compile and run as a non-secure application.
```

Comment:
```suggestion
When built for an ``_ns`` build target, the sample is configured to compile and run as a non-secure application.
```
I think that'll account for samples with multiple board options better. If we don't, then "for the \`\`_ns\`\` build target" would be better.